### PR TITLE
luac: add page

### PR DESCRIPTION
**Pull request by matusf**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/luac.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua byte code compiler.
+
+- Compile lua source code to lua byte code:
+
+`luac -o {{byte_code.luac}} {{source.lua}}`
+
+- Stop including debugging information in byte code:
```
Comment (stellarhopper) [inline pages/common/luac.md]: It probably reads better if you say "Strip debug information from the output" or "Don't include debug information in the byte code".  "Stop including.." implies some sort of a lasting effect..

Comment (matusf) [reply pages/common/luac.md]: I'd go for "Don't include debug information in the byte code"

File: pages/common/luac.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua byte code compiler.
```
Comment (sbrl) [inline pages/common/luac.md]: Not sure that there should be a space in `bytecode`.

File: pages/common/luac.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- Compile a lua source code to a lua byte code:
```
Comment (sbrl) [inline pages/common/luac.md]: Perhaps ...`Lua source file to Lua bytecode would read better?

File: pages/common/luac.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+`luac -o {{byte_code.luac}} {{source.lua}}`
+
+- Don't include a debug information in the byte code:
```
Comment (sbrl) [inline pages/common/luac.md]: Perhaps `Don't include debug information in the output` would read better?

File: pages/common/luac.md (hunk 2 above)
Comment (matusf) [reply pages/common/luac.md]: My spellchecker underlined it but on wiki it says bytecode. I'll change it

File: pages/common/luac.md (hunk 3 above)
Comment (matusf) [reply pages/common/luac.md]: I agree. I'll change it.

File: pages/common/luac.md (hunk 4 above)
Comment (matusf) [reply pages/common/luac.md]: Ok, but the article should probably stay there. So that it's grammatically correct. What do you think?

Comment (stellarhopper) [reply pages/common/luac.md]: 'information' is not a countable noun, so "a debug information" is incorrect.

File: pages/common/luac.md [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
-> Lua byte code compiler.
+> Lua bytecode compiler.
 
-- Compile a lua source code to a lua byte code:
+- Compile Lua source file to Lua bytecode:
 
 `luac -o {{byte_code.luac}} {{source.lua}}`
 
-- Don't include a debug information in the byte code:
+- Don't include a debug information in the output:
```
Comment (stellarhopper) [inline pages/common/luac.md]: 'information' is not a countable noun, so "a debug information" is incorrect.

File: pages/common/luac.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua bytecode compiler.
+
+- Compile Lua source file to Lua bytecode:
```
Comment (sbrl) [inline pages/common/luac.md]: Oops! Think I missed something here. Perhaps `Compile a Lua`.... would read better?

File: pages/common/luac.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua bytecode compiler.
+
+- Compile a Lua source file to Lua bytecode:
+
+`luac -o {{byte_code.luac}} {{source.lua}}`
+
+- Don't include debug information in the output:
```
Comment (agnivade) [inline pages/common/luac.md]: How about "Do not include debug symbols in the output:" ? So that we match the flag `-s` with a word in the description.

**Pull request merged**